src/app/pages/storage2/modules/devices/components/disk-info-card/disk-info-card.component.html

```diff
@@ -0,0 +1,75 @@
+<mat-card class="container-card">
+  <mat-card-header>
+    <h3 mat-card-title>{{ 'Disk Info' | translate }}</h3>
+    <button
+      mat-button
+      *ngIf="!!diskInfo"
+      (click)="onEdit()"
+    >{{ 'Edit' | translate }}</button>
+  </mat-card-header>
+  <mat-card-content>
+    <ng-container *ngIf="loading || !!diskInfo; else noDiskInfo">
+      <div class="row">
+        <span class="header">{{ 'Disk Size' | translate }}:</span>
+        <span *ngIf="!loading; else loaderSkel">
+          {{ diskInfo?.size ? formatter.convertBytesToHumanReadable(diskInfo.size) : 'N/A' }}
```

Comment:
`N/A` is not translated. Some example where `N/A` string is a template
https://github.com/truenas/webui/blob/eae180594660d6bcfed67b513f0350ed28edd68b/src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.html#L64
https://github.com/truenas/webui/blob/eae180594660d6bcfed67b513f0350ed28edd68b/src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.html#L120-L122